.dependabot/config.yml

```diff
@@ -0,0 +1,16 @@
+version: 1
+update_configs:
+- package_manager: "java:maven"
+  directory: "/"
+  update_schedule: "weekly"
+  default_reviewers:
+    - "alecharp"
+    - "varyvol"
+  ignored_updates:
+    - match:
+        dependency_name: "org.jenkins-ci.main:jenkins-core"
+    - match:
+        dependency_name: "org.jenkins-ci.plugins*"
```

Comment:
Why ignoring the plugins?

Reply:
I copied the configuration from `mailer` plugin and I checked with others (like `swarm` plugin) and they have the same settings. Maybe @oleg-nenashev would have more details.

Reply:
Just because we do not want to force all plugin updates when it not necessary . Up to plugin maintainers
Note that this configuration will also exclude Plugin POM which is under the same group ID

Reply:
so @varyvol, do you want to update the plugin pom as soon as a new one is published? If no, we can keep the configuration like that.